frontend/pages/hosts/details/cards/Software/HostSoftware.tsx

```diff
@@ -238,6 +241,28 @@ const SoftwareCard = ({
 
   const data = isMyDevicePage ? deviceSoftwareRes : hostSoftwareRes;
 
+  if (hostPlatform === "ios" || hostPlatform === "ipados") {
+    return (
+      <Card
+        borderRadiusSize="large"
+        includeShadow
+        largePadding
+        className={baseClass}
+      >
+        <p className="card__header">Software</p>
```

Comment:
Just nice to have and not blocking, it would be clean to encapsulate only the variable parts of the code (children of `p`) with the below option since `Card` and `p` share the same attributes.